### PR TITLE
DBZ-5878: Update the Debezium UI Documentation page to include the latest additional steps and UI screenshots

### PR DESCRIPTION
**Pull request by indraraj**
Closes https://issues.redhat.com/browse/DBZ-5878
Included the description of **Transformations**, **Topic creation** and **Custome properties** additional steps.
Updated the old Debezium UI screenshots.

### CODE REVIEW

**Comment by jpechane** [comment]
@rk3rn3r @roldanbob Could one of you take a look please? Thanks

File: documentation/modules/ROOT/pages/operations/debezium-ui.adoc [hunk 1]
```diff
@@ -89,23 +89,26 @@ image::debezium-ui-connectors-list.png[{prodname} UI connectors]
...
 
-image::debezium-ui-create-connector-step2.png[{prodname} UI Create connector step2]
+image::debezium-ui-step2.png[{prodname} UI Create connector step2]
 
 ==== Create Connector - Additional properties (optional)
 The Additional properties are optional and can be summarized as follows:
 
 * *Filter definition* - entry of *regular expressions* which define the filters for inclusion/exclusion of the items that will be included for CDC.  The included items are displayed as the filters are entered and applied.
+* *Transformations* - enable single message at a time modification. This can use to to modify records before they are sent to Apache Kafka by applying them to the {prodname} connectors. Read more about the transformation here xref:{link-transformations}[transformations]
+* *Topic creation* - define automatic topic creation default properties and groups.
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/operations/debezium-ui.adoc:106]:
```suggestion
* *Topic creation* - Define default properties and groups for enabling automatic topic creation.
```

File: documentation/modules/ROOT/pages/operations/debezium-ui.adoc [hunk 2]
```diff
@@ -89,23 +89,26 @@ image::debezium-ui-connectors-list.png[{prodname} UI connectors]
...
 * *Data options* - *Snapshot* and *Mapping* properties (optional).  The defaults can be viewed and changed if desired.
 * *Runtime options* - *Engine* and *Heartbeat* properties (optional).  The defaults can be viewed and changed if desired.
+* *Custom properties* - enable pass-thru of additional properties that are not covered by the previous sections can be entered here e.g database.* and database.history.* options. The user can enter any valid property name and value.
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/operations/debezium-ui.adoc:109]:
```suggestion
* *Custom properties* - Enable pass-through of additional properties that are not specified in the previous sections, such as `database.*` and `database.history.*` options. You can enter any valid property name and value.
```

**Review by roldanbob — CHANGES_REQUESTED** [review]
Generally, to avoid confusion about who should perform a task, try to address "the user" directly as "you" rather than in the third person. 
When referring to a UI label, always use bold text vs. quotation marks. Also, avoid the construction "click the <button-label> button". Just say, click <button-label>.

**Comment by jpechane** [comment]
@indraraj @roldanbob Applied, thanks!

File: documentation/modules/ROOT/pages/operations/debezium-ui.adoc [hunk 3]
```diff
@@ -79,7 +79,7 @@ A self-contained example https://github.com/debezium/debezium-examples/tree/main
 The main page of the UI displays all the registered connectors.  Some of the highlights of the main page are as follows:
 
 * *Kafka connect cluster* can be selected via the dropdown in the header.
-* Connector table shows each connector with it's type (MySQL, PostgreSQL, MongoDB), connector status and connector tasks.
+* Connector table shows each connector with it's type (MongoDB, MySQL, Oracle, PostgreSQL and SQL Server), connector status and connector tasks.
 * A connector row can be expanded to show more details, as shown below with the 'testPostgres' connector.  Metrics are shown in the expansion area (*Note:* this feature is still under development and not functional yet).  Connector tasks are shown, with ability to *Restart* the task if desired.
 * The kebab menu at the right of each connector row provides actions which allow the user to *Pause, Resume, Restart or Delete* the connector.
```
[the remark this answers is not in the document]
Comment (indraraj) [reply documentation/modules/ROOT/pages/operations/debezium-ui.adoc]: updated

**Pull request merged**